### DISCUSSION TITLE
Welcome to chatgpt4 Discussions!

### DISCUSSION THREAD

**Discussion by tianming12333** [tianming12333/chatgpt4#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by lamblife** [comment]
Hi I hope I can get some help. I have been using chatgpt4 for several months. Suddenly I can't get into my account. There may be some problem with the Google saving of my password and or email or both. Can anyone help? thanks
